internal/babysitter/babysitter.go

```diff
@@ -77,13 +78,13 @@ func NewBabysitter(
 	replicaSet string,
 	podName string,
 	useLocalhost bool,
-	caCert *x509.Certificate,
-	selfCertPEM []byte,
-	selfKeyPEM []byte,
 	manager clients.ManagerClient,
+	selfCertGetter func() ([]byte, []byte, error),
+	caCertGetter func() (*x509.Certificate, error),
```

Comment:
We chatted offline and discovered that CA certs are valid for 10 years and shouldn't be rotated.